examples/passport/wallets-connect-with-nextjs/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@examples/wallets-connect-with-nextjs",
   "version": "0.1.0",
   "dependencies": {
-    "@biom3/react": "^0.27.9",
+    "@biom3/react": "^0.27.10",
```

Comment:
plz ignore this, ill setup a seperate PR to bump biome. Lost a bit of time yesterday with biome update PR's - so just wanted to try it all out in this PR before making the final dep bump PR